diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/files/NewlinesRuleWarnTest.kt

```diff
@@ -527,6 +527,18 @@ class NewlinesRuleWarnTest : LintTestBase(::NewlinesRule) {
         )
     }
 
+    @Test
```

Comment:
create fixer tests (where `WRONG_NEWLINES` is fixing the code in such case) to prove that you are right

Reply:
@Cheshiriks where are tests for fixers?